components/event-bus/cmd/event-publish-service/application/application.go

```diff
@@ -56,7 +57,7 @@ func (app *KnativePublishApplication) start() {
 
 	// mark the app as started and register the readiness and the publish handlers
 	app.started = true
-	app.registerReadinessProbe()
+	app.registerReadinessProbe(app.knativeLib.EvClient())
```

Comment:
A check should ensure that `knativeLib != nil`

Reply:
Added